sentinel-extension/sentinel-annotation-aspectj/src/main/java/com/alibaba/csp/sentinel/annotation/aspectj/SentinelResourceAspect.java

```diff
@@ -64,7 +64,7 @@ public Object invokeResourceWithSentinel(ProceedingJoinPoint pjp) throws Throwab
                 throw ex;
             }
             if (exceptionBelongsTo(ex, annotation.exceptionsToTrace())) {
-                traceException(ex, annotation);
+                traceException(ex);//exceptionsToIgnore not include ex &&  exceptionsToTrace include ex
```

Comment:
Maybe the comment is not needed here?

Reply:
sorry to interrupt you,it is my first pr :( 
i create a new branch,and set git config --global user.email "307517019@qq.com",but it still include the first pr(at that time,i did not set the git config: email),so i cannot pass the cla check

Reply:
You may need to squash your commits with your correct email (you could leverage `git rebase` with squash mode).

Reply:
And you could just drop the last commit during your `rebase`.